app/controllers/provider_foreman_controller.rb

```diff
@@ -403,7 +423,7 @@ def show_record(_id = nil)
     end
 
     if @record.class.base_model.to_s == "ConfiguredSystem"
-      rec_cls = @record.class.base_model.to_s.underscore
+      rec_cls = "#{model_to_name(@record.class.to_s).downcase.tr(' ', '_')}_configured_system"
```

Comment:
the provider_foreman+rec_cls is used in constucting the name for the toolbar files in the provider accordion and the other ones for the toolbar files in the configured_systems accordion, Granted - they are the same now, but may contain different options in the future.